dcmgr/lib/dcmgr/rpc/windows_handler.rb

```diff
@@ -21,11 +21,18 @@ class WindowsHandler < EndpointBuilder
         [@hva_ctx]
       )
 
+      after_in_minutes = Dcmgr::Configurations.hva.windows.delete_password_after
+      if after_in_minutes == 0
+        expire = nil
+      else
+        expire = Time.now.utc + ( after_in_minutes * 60 )
+      end
+
```

Comment:
Fixed with https://github.com/axsh/wakame-vdc/commit/6d39a9a9eac11e9609a03e2035753d806db3725a

"Added validation for delete_password_after hva.conf parameter"